app/views/posts/index.html.erb

```diff
@@ -1,14 +1,18 @@
-<p style="color: green"><%= notice %></p>
-
-<h1>Posts</h1>
-
-<div id="posts">
-  <% @posts.each do |post| %>
-    <%= render post %>
-    <p>
-      <%= link_to "Show this post", post %>
-    </p>
-  <% end %>
-</div>
-
-<%= link_to "New post", new_post_path %>
+<div class="posts">
+    <img src="<%= @user.photo %>" />
+    <h1><%= @user.name %></h1>
+    <h2><%= @user.bio %></h2>
+    <p><%= @user.posts_counter %></p>
+    <% @posts.each do |post| %>
+            <div class="schow_posts">
+                <h3><%=post.title %></h3>
+                <p><%=post.text %></p>
+                <p>comment: <%=post.comments_counter %>
+                <p>likes: <%=post.likes_counter %>
+                <h3>Commetes</h3>
+                <% post.comments.each do |comment| %>
+                    <p><%=comment.text %></p>
+                <% end %>

```

Comment:
- [ ]  Following the project requirements, you should use `post.recent_comments` method to render 5 most recent comments in the `app/views/posts/index.html.erb` page.